packages/middleware-websocket/src/middleware-websocket-endpoint.spec.ts

```diff
@@ -42,7 +42,7 @@ describe(websocketEndpointMiddleware.name, () => {
     const request = new HttpRequest({
       headers: {
         "content-type": "application/vnd.amazon.eventstream",
-        "Content-Type": "application/vnd.amazon.eventstream",
+        "content-type": "application/vnd.amazon.eventstream",
```

Comment:
This is original case to test removal of both cases
```suggestion
        "Content-Type": "application/vnd.amazon.eventstream",
```